### ISSUE TITLE
Browse Series Title in Blacklight

### ISSUE THREAD

**Issue by minjiec** [pulibrary/orangelight#4659]
### Current display
Blacklight’s display of series titles does not fully conform to the MARC definition of which fields can serve as access points. It seems like–

- _440, 490 1#, 830 are hyperlinked_
- _800, 810, 811 are not_

Note that 490 "[does not serve as a series access point](https://www.oclc.org/bibformats/en/4xx/490.html)"

### Desired display in accordance with MARC definitions
The following fields can serve as series titles access points, thus be made into browsable hyperlinks in Blacklight:

- [440](https://www.loc.gov/marc/bibliographic/bd440.html) 
- [800](https://www.oclc.org/bibformats/en/4xx.html) personal name+title
- 810 corporate name+title
- 811 meeting name+title
- 830 Uniform Title

For field 490 (_transcribed_ series statements)–

1. Display it as plain text in Blacklight only
2. Continue indexing it so that its content is searchable as keywords from series titles

### Notes
Proposed by Minjie Chen, Yang Wang, Rex Hatfield

**Comment by maxkadel** [comment]
@caroldh @ellen-aa would you give some feedback on prioritization for this ticket?

**Comment by caroldh** [comment]
@maxkadel  - Ellen and I just talked about this - is there a sample record that illustrates the link/lack of link?

**Comment by maxkadel** [comment]
@minjiec  - would you be able to provide some examples to illustrate this issue?

**Comment by minjiec** [comment]
Yes, @maxkadel , these are some of the [examples](https://docs.google.com/document/d/1QzsWwTSf2x4WYASP49CG72wjWQ7KdM2EZ57IbV--4ec/edit?tab=t.0#heading=h.2w1ta1rd0jhh) Yang, Rex and I examined to come up with the proposal. Does it help?
